python/paddle/v2/fluid/layers/nn.py

```diff
@@ -64,7 +66,11 @@
     'nce',
     'beam_search',
     'row_conv',
+    'reshape',
```

Comment:
`reshape` has been exposed to Python in https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/layers/ops.py#L50

Reply:
Done